packages/ui/src/universal/font-analyzer/index.tsx

```
@@ -0,0 +1,82 @@
+        }
+        const entry = usage.get(weight)!;
+        entry.components.add(component);
+        entry.count++;
+      };
+
+      // Analyze all elements
+      document.querySelectorAll('*').forEach(element => {
+        const computedStyle = window.getComputedStyle(element);
+        const fontWeight = parseInt(computedStyle.fontWeight);
+
+        if (!isNaN(fontWeight)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)